rmgpy/rmg/model.py

```diff
@@ -1928,9 +1928,12 @@ def generateReactionKey(rxn, useProducts=False):
     key_family = rxn.family
 
     spc_list = rxn.products if useProducts else rxn.reactants
-    key1 = getKey(spc_list[0])
-    key2 = None if len(spc_list) == 1 else getKey(spc_list[1])
-    key1, key2 = sorted([key1, key2], reverse=True)  # ensure None is always at end
+
+    if len(spc_list) == 1:
+        key1, key2 = getKey(spc_list[0]), None
+    else:
+        # Use 2 keys, even for trimolecular reactions?
```

Comment:
We definitely supported trimolecular library reactions at the time this was written, so it might have been considered. I feel like it would be more efficient to have 3 keys for trimolecular. With that said, I think it's probably not a good idea to change this now, and instead revisit it after py3.